src/components/ApiKeyInput/ApiKeyInput.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 import styles from './ApiKeyInput.module.css';
-import validateApiKey from '../../utils/validation.js';
+import validateApiKey from '../../../netlify/utils/validations.js';
```

Comment:
Have you tested to make sure this works? I think it will break because you are trying to import from outside the `src` folder.

A better way is to symlink netlify's validation `utils` to the frontend. I have left a more indepth explanation of what I mean by that in this PR :) https://github.com/SyedArslanHaider/smart-cv-builder/pull/124

Reply:
with the symlink we need to remove one of these directory traversals

ie

```jsx
import validateApiKey from '../../netlify/utils/validations.js';
```